os/board/rtl8730e/src/component/bluetooth/example/ble_scatternet/scatternet.c

```diff
@@ -295,7 +295,7 @@ static rtk_bt_evt_cb_ret_t ble_tizenrt_scatternet_gap_app_callback(uint8_t evt_c
         if (!conn_ind->err) {
```

Comment:
Pleaes check commit title

[This is a commit rule](https://github.com/Samsung/TizenRT/wiki/Documentations#commit-rules)